Base/Python/slicer/tests/test_slicer_parameter_node_wrapper.py

```diff
@@ -525,6 +531,102 @@ class ParameterNodeType:
         param2 = ParameterNodeType(param.parameterNode)
         self.assertEqual(param2.p, [[4, 5, 0, 7]])
 
+    def test_tuple(self):
+        @parameterNodeWrapper
+        class ParameterNodeType:
+            a: tuple[Annotated[int, Minimum(0)], str]
+            b: Annotated[tuple[float, bool], Default((44.0, False))]
```

Comment:
> I did not think an `ObservedTuple` was needed due to the immutability of `tuple`s.

There is still the case that one of the tuple's elements is mutable; it might be worth adding something like `tuple[list[int]]` to the test suite. I wouldn't expect issues since the logic is the same as other containers.

Reply:
Added test for tuple of lists (which incidentally is also a test for a 1 value tuple).